src/modules/SX126x/SX126x.h

```diff
@@ -827,7 +827,9 @@ class SX126x: public PhysicalLayer {
    float getRSSIInst();
 
    /*!
-     \brief Set implicit header mode for future reception/transmission.
+     \brief Set implicit header mode for future reception/transmission. Required for spreading factor 6.
+
+     \param len Payload length in bytes.
```

Comment:
Implicit header is not required for SF6 on SX126x modules. In fact, SX127x FS6 is incompatible with SX126x.

Reply:
Sorry. Was doing a search and replace.